assets/src/components/propertiesPanel/minischedule.tsx

```diff
@@ -630,16 +630,16 @@ const iconForDirectionOnLadder: (
 ) => ReactElement = (directionId, ladderDirections, routeId) => {
   const iconClassName = "m-minischedule__svg--revenue"
   if (directionId === null) {
-    return questionMarkIcon(iconClassName)
+    return QuestionMarkIcon({ className: iconClassName })
```

Comment:
**question:** can this and the uses of `Triangle[Up|Down]Icon` below be written in JSX syntax?

Reply:
Question or request?
It can, I was attempting to preserve the old functionality of using a function call. But this can be changed!

Reply:
Hah... I guess more of a request since I personally find the JSX syntax easier in that it quickly tells me that the thing I'm looking at is a component. I'm open to pushback on that though.